bin/foreman-node

```diff
@@ -162,21 +162,21 @@ def enc(minion)
   http             = Net::HTTP.new(uri.host, uri.port)
   http.use_ssl     = uri.scheme == 'https'
   if http.use_ssl?
-    if SETTINGS[:ssl_ca] && !SETTINGS[:ssl_ca].empty?
+    if SETTINGS[:ssl_ca].present?
       http.ca_file = SETTINGS[:ssl_ca]
       http.verify_mode = OpenSSL::SSL::VERIFY_PEER
     else
       http.verify_mode = OpenSSL::SSL::VERIFY_NONE
     end
-    if SETTINGS[:ssl_cert] && !SETTINGS[:ssl_cert].empty? && SETTINGS[:ssl_key] && !SETTINGS[:ssl_key].empty?
+    if SETTINGS[:ssl_cert].present? && SETTINGS[:ssl_key].present?
       http.cert = OpenSSL::X509::Certificate.new(File.read(SETTINGS[:ssl_cert]))
       http.key  = OpenSSL::PKey::RSA.new(File.read(SETTINGS[:ssl_key]), nil)
     end
   elsif SETTINGS[:username] && SETTINGS[:password]
     req.basic_auth(SETTINGS[:username], SETTINGS[:password])
   end
 
-  res = http.start { |http| http.request(req) }
+  res = http.start { |httpo| httpo.request(req) }
```

Comment:
Maybe it's more like a connection so `conn` or `connection`?
```suggestion
  res = http.start { |conn| conn.request(req) }
```

Reply:
Thanks.